scripts/build/rollup-plugin-javascript-prefixer.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable import/no-extraneous-dependencies */
+const glob = require('glob');
+const fs = require('fs').promises;
+
+const componentPackageJson = require(`${process.cwd()}/package.json`);
+
+const JS_FILES_TO_PREFIX = glob
+  .sync('**/*.js', {
+    ignore: [
+      'node_modules/**/*',
+      'dist/**/*',
+      'lib/**/*',
+      'index.react.js',
+      '**demo.js',
+      '**.bak',
+      '**test**',
+      '**story**',
+    ],
+  })
+  .map(f => `./${f}`);
+
+const _getComponentInfo = () => {
+  const elementMapping = new Map();
+  elementMapping.set('10-atoms', 'a-');
+  elementMapping.set('20-molecules', 'm-');
+  elementMapping.set('30-organisms', 'o-');
+
+  const cwdAsStringArray = process.cwd().split('/');
+  const componentDirectoryName = cwdAsStringArray[cwdAsStringArray.length - 2];
+  const customElementName = componentPackageJson.name.replace('@axa-ch/', '');
+  // atom (a-) / molecule (m-) / organism (o-)
+  const customElementTypePrefix = elementMapping.get( componentDirectoryName ); 
+  const newPrefix = `nva${componentPackageJson.version.replace(/\./g, '-')}`;
+  return {
+    prefix: newPrefix, // 'nva1-2-3'
+    standardComponentClassPrefix: customElementTypePrefix + customElementName, // a-button-link
+    cssPrefix: `${newPrefix}_${customElementTypePrefix}${customElementName}`, // .nva1-2-3_a-button-link
+  };
+};
+
+export const componentInfo = _getComponentInfo();
+
+/**
+ * Look for all classes and replace all that start with the BEM prefix and add
+ * the versioned prefix.
+ *
+ * @param {string} file Filename, e.g. './index.js'
+ */
+const _prefixJS = async file => {
+  await fs.copyFile(file, `${file}.bak`);
```

Comment:
@raphaellueckl if you have to wait anyway, better use SYNC functions of NODE. Is safer

Reply:
This method is obsolete now.